### ISSUE TITLE
Error log (classic)

### ISSUE THREAD

**Issue by Finrodi** [sylvanaar/prat-3-0#35]
```
1x ...ddOns\Prat-3.0-3.7.41_Libraries\LibSink-2.0\LibSink-2.0-90104.lua:438: bad argument #2 to 'format' (string expected, got table)
[C]: ?
...ddOns\Prat-3.0-3.7.41_Libraries\LibSink-2.0\LibSink-2.0-90104.lua:438: in function `generator'
...ddOns\Prat-3.0-3.7.41_Libraries\LibSink-2.0\LibSink-2.0-90104.lua:590: in main chunk

Locals:
(*temporary) = "Route output from this addon through %s."
(*temporary) = <unnamed> {
 0 = <userdata>
}
(*temporary) = "string expected, got table"

```

& 

```
1x Prat-3.0-3.7.41\addon\modules.lua:262: Usage: EmbedLibrary(addon, libname, silent, offset): 'libname' - Library 'LibSink-2.0-90104' is not Embed capable
Prat-3.0-3.7.41\addon\addon.lua:237: in function <Prat-3.0\addon\addon.lua:191>
[C]: ?
...\common\Wildpants\libs\AceAddon-3.0\AceAddon-3.0-12.lua:70: in function <...\common\Wildpants\libs\AceAddon-3.0\AceAddon-3.0.lua:65>
...\common\Wildpants\libs\AceAddon-3.0\AceAddon-3.0-12.lua:498: in function `InitializeAddon'
...\common\Wildpants\libs\AceAddon-3.0\AceAddon-3.0-12.lua:613: in function <...\common\Wildpants\libs\AceAddon-3.0\AceAddon-3.0.lua:605>
```

**Comment by sylvanaar** [comment]
I am unable to reproduce this. Do you have any saved variables for your customfilters module?

**Comment by Finrodi** [comment]
> I am unable to reproduce this. Do you have any saved variables for your customfilters module?

Not sure what you mean. You want me to upload my savedvariables files? :p

**Comment by Finrodi** [comment]
Whenever I go to "module control" the cursor bugs out, also I cant move the window properly, it resets and glitches. Dunno if that helps. I did delete saved variables for a clean reset but it didnt help.

**Comment by Finrodi** [comment]
https://drive.google.com/uc?id=1t05U9x38oLkCCiKHSsC7O3vlLsqRWqZZ video file of it.

**Comment by Mitalie** [comment]
Looks like LibSink-2.0 (required by Prat) was failing to load because it relied on strings in some Blizzard-created global variables, and some other addon is overwriting those globals with non-strings. While the other addon might be breaking something else too, LibSink-2.0 was fixed in https://www.curseforge.com/wow/addons/libsink-2-0/files/2836736.

A fixed version of LibSink-2.0 should be automatically included in future Prat builds.

**Comment by sylvanaar** [comment]
Thanks @amakinen that was really helpful!

@Finrodi  I will trigger a new build
